app/modules/handler/containers/Prompt.js

```diff
@@ -53,12 +54,18 @@ class PromptContainer extends Component<Props> {
     actions.signURI(tx, blockchain, wallet);
   }
   templateURI = () => {
-    const { blockchains, prompt, wallets } = this.props;
-    const { chainId } = prompt;
+    const { blockchains, prompt: oldprompt, wallets } = this.props;
```

Comment:
Why did you rename prompt?

Reply:
Renamed the variable while testing and forgot to revert the change.